tensorboard/components/tensor_widget/dtype-utils-test.ts

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {expect} from 'chai';
+
+import {isIntegerDType, isFloatDType} from './dtype-utils';
+
+describe('isIntegerDType', () => {
+  it('returns true for unsigned ints', () => {
```

Comment:
thanks for making the changes. 

Just wondering (I know the answer is 'yes', reading the code), but should this function return true for `int0` or `int2131232`?

Reply:
Yes, it will. But if there is actually a dtype called `int0`, it should be reasonably categorized as an integer. What other dtype category may make more sense? Same for `int2131232`.

Reply:
Adding one assertion around it will make me understand the behavior better :)

Reply:
Following offline discussion, I'm adding a few more assertions, but I'm skipping unrealistic cases such as 'int0' for now.